app/src/main/java/org/simple/clinic/facility/change/confirm/ConfirmFacilityChangeSheet.kt

```diff
@@ -0,0 +1,30 @@
+package org.simple.clinic.facility.change.confirm
+
+import android.os.Bundle
+import org.simple.clinic.ClinicApp
+import org.simple.clinic.R
+import org.simple.clinic.facility.change.confirm.di.ConfirmFacilityChangeComponent
+import org.simple.clinic.widgets.BottomSheetActivity
+
+class ConfirmFacilityChangeSheet : BottomSheetActivity() {
+
+  companion object {
+    lateinit var component: ConfirmFacilityChangeComponent
+  }
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+
+    setContentView(R.layout.sheet_confirm_facility_change)
+    setupDi()
```

Comment:
~I think we are supposed to do it in `attachBaseContext`, can you PTAL?~ 
NVM it has been added in a later commit.

Reply:
Yea. I forgot that the first time.